cmake.patch

```diff
@@ -2,7 +2,18 @@ diff --git a/CMakeLists.txt b/CMakeLists.txt
 index 004a2a9..bd87433 100644
 --- a/CMakeLists.txt
 +++ b/CMakeLists.txt
-@@ -88,5 +88,6 @@ if(GLAD_INSTALL)
+@@ -76,6 +76,10 @@
+   set_target_properties(glad PROPERTIES LINKER_LANGUAGE ${GLAD_LINKER_LANGUAGE})
+ endif()
+ 
++if(CMAKE_GENERATOR MATCHES "Visual Studio" AND BUILD_SHARED_LIBS)
```

Comment:
I don't think condition should rely on CMake generator. you can build via Ninja or NMake generators, for instance. so, it probably should be just `if (MSVC)`

Reply:
Is it worth doing something like `if (MSVC AND BUILD_SHARED_LIBS)` or is that overkill?

Reply:
yes, `if (MSVC AND BUILD_SHARED_LIBS)` is perfect fit here